crates/runix/src/command.rs

```diff
@@ -269,3 +270,24 @@ impl JsonCommand for PathInfo {}
 impl TypedCommand for PathInfo {
     type Output = Vec<Narinfo>;
 }
+
+/// `nix store gc` Command
+#[derive(Debug, Clone)]
+pub struct StoreSign {
+    /// `store sign` supports some extended installable options
+    /// which are also available elsewhere and not yet formalized.
```

Comment:
It seems like this comment is just talking about `--recursive`, so it would probably be more clear if it was concrete, just saying `--recursive` instead of `some extended installable options`. Maybe there are more I'm unaware of though

Reply:
there is some more: `--all`, `--derivation` and `--recursive` are additional installable options in some cases


Nix store sign (ans some others, e.g. **copy**)
```
    Options that change the interpretation of installables:

      · --all

        Apply the operation to every store path.

      · --derivation

        Operate on the store derivation rather than its outputs.

      · --expr expr

        Interpret installables as attribute paths relative to the Nix expression expr.

      · --file / -f file

        Interpret installables as attribute paths relative to the Nix expression stored in file. If file is the character -, then a Nix expression will be read from standard input. Implies --impure.

      · --recursive / -r

        Apply operation to closure of the specified paths.
```

vs 

```
    Options that change the interpretation of installables:

      · --expr expr

        Interpret installables as attribute paths relative to the Nix expression expr.

      · --file / -f file

        Interpret installables as attribute paths relative to the Nix expression stored in file. If file is the character -, then a Nix expression will be read from standard input. Implies --impure.

```

Reply:
```suggestion
    /// `store sign` (and some other commands) support additional installable options
    ///  `--all`, `--derivation` and `--recursive` 
    /// which are not yet formalized.
```